app/components/listing/ServiceAttribution.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import ServiceVerified from '../../assets/img/ic-attributed-record.svg';
+
+import './ServiceAttribution.scss';
+
+class ServiceAttribution extends React.Component {
+  isServiceNetAttribution() {
+    const { attribution, status } = this.props;
+    return attribution === 'service_net' && status === 'approved';
+  }
+
+  render() {
+    const benetechLink = 'https://medium.com/@Shelter_Tech/sheltertech-is-participating-'
+      + 'in-the-benetech-service-net-pilot-in-the-san-francisco-bay-area-b28645d3dee6';
+
+    return this.isServiceNetAttribution() ? (
```

Comment:
Nit: I think usually we prefer to use the `predicate && <JSX stuff>` pattern over `predicate ? <JSX Stuff> : null` when the "else" case should be to return nothing.

Hopefully you don't mind, but I'll make that minor tweak before merging this in.